pom.xml

```diff
@@ -181,7 +181,13 @@
         <cronstrue.version>1.79.0</cronstrue.version>
         <!--other properties-->
         <skip.integration.tests>true</skip.integration.tests>
-        <scalastyle.configLocation>${project.basedir}/scalastyle-config.xml</scalastyle.configLocation>
+        <!--suppress UnresolvedMavenProperty -->
```

Comment:
This seems dangerous. Is this because of the sub-module poms?

Reply:
Added during development, missed it to remove. Will delete it.